source/guides/installfest/admin_and_markdown.md

```diff
@@ -6,151 +6,166 @@ github_url: https://github.com/reinteractive-open/installfest_guides/tree/master
 
 ## Adding ActiveAdmin and Markdown support to Rails
 
-Welcome back to reInteractive's Ruby on Rails 15 minute blog tutorial series. If you haven't started following through the series and you're new to Rails then you might want to start with the [first post](/guides/installfest/getting_started). Today we'll be following directly on from [Part 3](/guides/installfest/testing_the_blog). If you feel confident with Rails but want to learn more about ActiveAdmin and/or Markdown you can find some instructions on getting the code set up properly below.
+Welcome back to reInteractive's Ruby on Rails 15 minute blog tutorial series.
+If you haven't started following through the series and you're new to Rails
+then you might want to start with the [first
+post](/guides/installfest/getting_started). Today we'll be following directly
+on from [Part 3](/guides/installfest/testing_the_blog). If you feel confident
+with Rails but want to learn more about ActiveAdmin and/or Markdown you can
+find some instructions on getting the code set up properly below.
 
 ## Installing an Administration System
 
-One of the big problems with our blog is that we're using HTTP Basic Authentication to prevent anyone from creating and editing blog posts and that the links to perform these actions are right there in the blog. Instead we'd prefer to have an admin panel where we could manage our blog posts. We could build one from scratch but there's a fantastic gem called [ActiveAdmin](https://github.com/gregbell/active_admin) which we can use to easily give us what we want.
+One of the big problems with our blog is that we're using HTTP Basic
+Authentication to prevent anyone from creating and editing blog posts and that
+the links to perform these actions are right there in the blog. Instead we'd
+prefer to have an admin panel where we could manage our blog posts. We could
+build one from scratch but there's a fantastic gem called
+[ActiveAdmin](https://github.com/activeadmin/activeadmin) which we can use to
+easily give us what we want.
 
 ### Application Setup
 
-You'll need to have been following our InstallFest blog posts starting with [http://reinteractive.net/posts/32](/guides/installfest/getting_started) and have completed [http://reinteractive.net/posts/42](/guides/installfest/testing_the_blog). If you've done this but want to start with some fresh code, you can by copying the tag that's available in the public git repository.
+You'll need to have been following our InstallFest blog posts starting with
+[http://reinteractive.net/posts/32](/guides/installfest/getting_started) and
+have completed
+[http://reinteractive.net/posts/42](/guides/installfest/testing_the_blog).
```

Comment:
This one to https://reinteractive.net/posts/42-testing-your-blog-in-15-minutes-part-3

both links a bit outdated...